source/django-compatibility.txt

```diff
@@ -0,0 +1,58 @@
+.. _django-compatibility:
+
+=============
+Compatibility
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: backwards compatibility, versions, upgrade 
+
+MongoDB Compatibility
+---------------------
+
+{+django-odm+} uses the PyMongo driver to interact with MongoDB. The {+mdb-server+}
+versions that your application supports depend on your PyMongo version.
+The following compatibility table specifies the recommended version or versions
+of PyMongo for use with a specific version of MongoDB.
+
+The first column lists the driver version.
+
+.. sharedinclude:: dbx/lifecycle-schedule-callout.rst
+
+.. sharedinclude:: dbx/compatibility-table-legend.rst
+
+.. include:: /includes/mongodb-compatibility-table-pymongo.rst
```

Comment:
Are we required to list the PyMongo<>MongoDB compatibility information here or can we link to it instead? At a glance, it looks overwhelming, and the most important message to convey is Django 5.x == Django MongoDB 5.x.

Reply:
I'm fine just linking to it!